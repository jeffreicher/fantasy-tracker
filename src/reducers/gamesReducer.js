import { FETCH_GAMES } from '../actions'
export default function(state = [], action) {
  // console.log(action.type)
  switch (action.type) {
    case FETCH_GAMES:
      return {
        ...state,
        games: action.payload,
        noGames: false
      }
    default:
      return state
  }
}
