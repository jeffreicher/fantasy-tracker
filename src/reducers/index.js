import { combineReducers } from 'redux';

const rootReducer = combineReducers({
    // games: GamesReducer
});

export default rootReducer;