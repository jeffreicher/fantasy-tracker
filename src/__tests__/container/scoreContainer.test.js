import React from 'react'
import ReactDOM from 'react-dom'
import ScoreContainer from '../../container/scoreContainer'
import { Provider } from 'react-redux'
import reducers from '../../reducers'
import ReduxPromise from 'redux-promise'
import { createStore, applyMiddleware } from 'redux'
import renderer from 'react-test-renderer'

const createStoreWithMiddleware = applyMiddleware(ReduxPromise)(createStore)
describe('tests Score Container component', () => {
  it('matches Score Container snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={createStoreWithMiddleware(reducers)}>
          <ScoreContainer />
        </Provider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
