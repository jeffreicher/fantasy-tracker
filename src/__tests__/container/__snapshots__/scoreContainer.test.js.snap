// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests Score Container component matches Score Container snapshot 1`] = `
<div>
  NO GAMES FOR TODAY.
</div>
`;
